ui-core/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,11 @@
+<!--
+  ~ Copyright (c) 2019 Adyen N.V.
+  ~
+  ~ This file is open source and available under the MIT license. See the LICENSE file for more info.
+  ~
+  ~ Created by caiof on 21/3/2019.
```

Comment:
Huh why is not your name and the wrong date here?

Reply:
Copy-paste errors. Fixed it. 😅